.github/workflows/go.yaml

```diff
@@ -71,6 +72,21 @@ jobs:
         # root ownership so the tests must be run as root:
         run: sudo make go.unit
 
+      - name: dqlite-for-golangci-lint
+        working-directory: src/k8s
+        run: |
+          sudo add-apt-repository ppa:dqlite/dev
+          sudo apt update
+          sudo apt install dqlite-tools libdqlite-dev
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v1.61
+          working-directory: src/k8s
+          args: '-c ./../../golangci.yml'
```

Comment:
Move this file to the go source root, as discussed. They you don't need to do this relative path dance.